README.md

```diff
@@ -23,41 +23,69 @@ The server component to the TourTrak system built using Django.
 * rabbitmq-server
 
 
-## Install PostGIS
+## Install Postgres with PostGIS (Install The Database)
 
-1. Install postgreSQL. Refer to official documentation. After, install postGIS, which is a postgreSQL extention for handling spatial data by doing `apt-get install postgresql-9.1-postgis`
-2. Setup the postgreSQL database:
+<i>(This is where you begin the server setup instructions)</i>
 
-* switch to the default postgresql user by running `su postgres`
-* create a user w/ read and write permissions: `createuser --pwprompt`
-* Create the DCS database used to collect rider information: `createdb DCS`
+1. Install postgreSQL
+
+	`sudo apt-get install postgresql`
+2. After, install postGIS, which is a postgreSQL extention for handling spatial data
+
+	`sudo apt-get install postgresql-9.3-postgis-2.1`
+3. Setup the postgreSQL database:
+
+* switch to the default postgresql user by running
+
+	`sudo -iu postgres`
+* create a user w/ read and write permissions:
+
+	`createuser --pwprompt`
```

Comment:
`createuser: creation of new role failed: ERROR: role "postgres" already exists`